# HG changeset patch
# User Rik <rik@octave.org>
# Date 1482811273 28800
#      Mon Dec 26 20:01:13 2016 -0800
# Node ID f9e6259c5fc1bfefc01a25a287ea6aef52a1847d
# Parent  3c72c72233e3001ea20b06712769a37b95f157df
decic.m: Fix bad @seealso macro from cset 80ac3e38b03d.

* decic.m: Fix bad @seealso macro from cset 80ac3e38b03d.

diff --git a/scripts/ode/decic.m b/scripts/ode/decic.m
--- a/scripts/ode/decic.m
+++ b/scripts/ode/decic.m
@@ -76,17 +76,17 @@
 ##   r = [-(@var{yp}(1) + 0.04*@var{y}(1) - 1e4*@var{y}(2)*@var{y}(3));
 ##        -(@var{yp}(2) - 0.04*@var{y}(1) + 1e4*@var{y}(2)*@var{y}(3) + 3e7*@var{y}(2)^2);
 ##        @var{y}(1) + @var{y}(2) + @var{y}(3) - 1];
 ## endfunction
 ## @end group
 ## [@var{y0_new},@var{yp0_new}] = decic (@@robertsidae, 0, [1; 0; 0], [1; 1; 0],
 ## [-1e-4; 1; 0], [0; 0; 0]);
 ## @end example
-## @seealso{ode1
+## @seealso{ode15i, odeset}
 ## @end deftypefn
 
 function [y0_new, yp0_new, resnorm] = decic (fun, t0,
                                              y0, fixed_y0, yp0, fixed_yp0,
                                              options)
 
   if (nargin < 6 || nargin > 7)
     print_usage ();
